src/game/behaviors/fish.inc.c

```diff
@@ -1,163 +1,299 @@
-// fish.c.inc
+/**
+ * @file fish.inc.c
+ * Implements behaviour and spawning for fish located in the Secret Aquarium and other levels.
+ */
 
-void ActionFish0(void) {
+/**
+ * Spawns fish with settings chosen by the field o->oBehParams2ndByte.
+ * These settings are animations, colour, and spawn quanitity.
+ * Fish spawning restricted to a set distance from Mario.
+ */
+void fish_act_spawn(void) {
     s32 i;
-    s32 sp38;
+    s32 schoolQuantity;
     s16 model;
-    f32 sp30;
-    struct Animation **sp2C;
-    struct Object *sp28;
+    f32 minDistToMario;
+    struct Animation **fishAnimation;
+    struct Object *fishObject;
     switch (o->oBehParams2ndByte) {
+        
+        // Blue fish with a quanitiy of twenty.
         case 0:
-            model = MODEL_FISH;            sp38 = 20;            sp30 = 1500.0f;            sp2C = blue_fish_seg3_anims_0301C2B0;
+            model = MODEL_FISH;
+            schoolQuantity = 20;
+            minDistToMario = 1500.0f;
+            fishAnimation = blue_fish_seg3_anims_0301C2B0;
             break;
+            
+        // Blue fish with a quanitiy of five.
         case 1:
-            model = MODEL_FISH;            sp38 = 5;            sp30 = 1500.0f;            sp2C = blue_fish_seg3_anims_0301C2B0;
+            model = MODEL_FISH;
+            schoolQuantity = 5;
+            minDistToMario = 1500.0f;
+            fishAnimation = blue_fish_seg3_anims_0301C2B0;
             break;
+            
+        // Cyan fish with a quanitiy of twenty.
         case 2:
-            model = MODEL_CYAN_FISH;            sp38 = 20;            sp30 = 1500.0f;            sp2C = cyan_fish_seg6_anims_0600E264;
+            model = MODEL_CYAN_FISH;
+            schoolQuantity = 20;
+            minDistToMario = 1500.0f;
+            fishAnimation = cyan_fish_seg6_anims_0600E264;
```

Comment:
NO NO NO you have to undo this. These lines need to be on the same line for PAL version to match. Same for all the other cases.

Reply:
But line limit of 104. (Didn't see this yesterday, will fix)